exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -2,190 +2,194 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using System.Transactions;
 
 namespace TriviaGame
 {
     public class Game
     {
-        List<string> players = new List<string>();
+        readonly List<string> players = new List<string>();
 
-        int[] places = new int[6];
-        int[] purses = new int[6];
+        readonly int[] places = new int[6];
+        readonly int[] coins = new int[6];
 
-        bool[] inPenaltyBox = new bool[6];
-
-        LinkedList<string> popQuestions = new LinkedList<string>();
-        LinkedList<string> scienceQuestions = new LinkedList<string>();
-        LinkedList<string> sportsQuestions = new LinkedList<string>();
-        LinkedList<string> rockQuestions = new LinkedList<string>();
+        readonly bool[] inPenaltyBox = new bool[6];
+              
+        readonly LinkedList<string> popQuestions = new LinkedList<string>();
+        readonly LinkedList<string> scienceQuestions = new LinkedList<string>();
+        readonly LinkedList<string> sportsQuestions = new LinkedList<string>();
+        readonly LinkedList<string> rockQuestions = new LinkedList<string>();
 
         int currentPlayer = 0;
         bool isGettingOutOfPenaltyBox;
 
         public Game()
         {
-            for (int i = 0; i < 50; i++)
+            for (int index = 0; index < 50; index++)
             {
-                popQuestions.AddLast("Pop Question " + i);
-                scienceQuestions.AddLast(("Science Question " + i));
-                sportsQuestions.AddLast(("Sports Question " + i));
-                rockQuestions.AddLast(CreateRockQuestion(i));
+                popQuestions.AddLast($"Pop Question {index}");
+                scienceQuestions.AddLast(($"Science Question {index}"));
+                sportsQuestions.AddLast(($"Sports Question {index}"));
+                rockQuestions.AddLast(CreateRockQuestion(index));
             }
         }
 
         public string CreateRockQuestion(int index)
         {
-            return "Rock Question " + index;
+            return $"Rock Question {index}";
         }
 
-        public bool IsPlayable()
-        {
-            return (HowManyPlayers() >= 2);
-        }
+        public bool IsPlayable() => HowManyPlayers() >= 2;
```

Comment:
I like these single line methods

Reply:
I had a discussion with Bryn yesterday on whether some of these could be C# properties:

public bool IsPlayable { get => HowManyPlayer() >= 2; }

I'm not sure of the answer but I though it might be worth considering, ultimately I don't think it matters too much. Bryn says a property might be better if it is something that belongs only to the class so IsPlayable might come under that.